Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -473,6 +474,7 @@ void Gui::MainWindow::editRO() {
     {
         m_materialEditor->changeRenderObject( item.m_roIndex );
         m_materialEditor->show();
+        // \fixme check here autoupdate
```

Comment:
I'ts the job of the m_materialEditor to ask for update if needed. There is nothing to add here.